src/components/Search/Search.scss

```diff
@@ -2,40 +2,47 @@
 
 .wkp-search {
   position: absolute;
-  top: 25px;
+  top: 26px;
   z-index: 2; // To be above header div, in order to be clickable.
   right: 320px;
   left: 400px;
 
+  .wkp-search-toggle-container {
+    .wkp-search-input {
+      background-color: white;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border: 2px solid #b7b7b7;
+    }
+  }
+
   .wkp-search-button {
     border: none;
     color: white;
-    height: 40px;
+    height: 42px;
```

Comment:
now you can use height:100%  isn't it ?

Reply:
I fixwd this in the meantime by making the button flex containers with height:100%, so the svgs align without padding/margin